packages/tailwindcss-x/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "target": "es2020",
+    "useDefineForClassFields": true,
+    "module": "commonjs",
+    "lib": ["ESNext", "DOM"],
+    "moduleResolution": "Node",
+    "strict": true,
+    "sourceMap": true,
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true,
+    "skipLibCheck": true,
+    "importHelpers": true,
```

Comment:
Enabling importHelpers, now we need to install tslib as dependency. See https://www.typescriptlang.org/tsconfig#importHelpers